internal/proxy/task_delete.go

```diff
@@ -413,21 +427,33 @@ func (dr *deleteRunner) Run(ctx context.Context) error {
 	return nil
 }
 
+func (dr *deleteRunner) getDeleteMsgPartitionID() int64 {
```

Comment:
The `partitionID` can be calculated only once per each `deleteRunner`, it would be better if the piece of code be executed in some initialization path, e.g., in `Init`